src/Symfony/Bridge/Doctrine/Tests/Types/UlidTypeTest.php

```diff
@@ -44,7 +44,7 @@ protected function setUp(): void
         $this->type = Type::getType('ulid');
     }
 
-    public function testUlidConvertsToDatabaseValue(): void
+    public function testUlidConvertsToDatabaseValue()
```

Comment:
why to remove the `void` return type?

Reply:
Because test cases don't define an API; they don't need to declare their signature. In the end, it's pure visual noise.